spec/features/course/achievement_listing_spec.rb

```diff
@@ -14,27 +12,25 @@
 
     before do
       login_as(user, scope: :user)
-    end
-
-    before do
       visit course_achievements_path(course)
     end
 
-    context 'management buttons' do
-      it { is_expected.to have_link(nil, href: new_course_achievement_path(course)) }
-    end
+    context 'As an Administrator' do
+      scenario 'I can create achievements' do
+        expect(page).to have_link(nil, href: new_course_achievement_path(course))
+      end
 
-    it 'shows all achievements' do
-      achievements.each do |achievement|
-        expect(page).to have_selector('div', text: achievement.title)
-        expect(page).to have_selector('div', text: achievement.description)
+      scenario 'I can view achievements' do
+        achievements.each do |achievement|
+          expect(page).to have_content_tag_for(achievement)
+        end
       end
-    end
 
-    it 'shows all management buttons' do
-      achievements.each do |achievement|
-        expect(page).to have_link(nil, href: edit_course_achievement_path(course, achievement))
-        expect(page).to have_link(nil, href: course_achievement_path(course, achievement))
+      scenario 'I can manage achievements' do
```

Comment:
in a future PR, merge these two spec files together... listing is a subset of management.